embedded.go

```diff
@@ -0,0 +1,22 @@
+package starboard
+
+import (
+	_ "embed"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/scheme"
+)
+
+var (
+	//go:embed deploy/crd/vulnerabilityreports.crd.yaml
```

Comment:
We cannot use relative path that start with . or ./.., therefore this embedded.go is in the top level starboard package